components/brave_wallet_ui/components/extension/panel-header-slim/index.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+
+// Styled Components
+import {
+  HeaderTitle,
+  HeaderWrapper,
+  TopRow,
+  CloseButton
+} from './style'
+// import { getLocale } from '../../../../common/locale'
+import { PanelTypes } from '../../../constants/types'
+
+export interface Props {
+  title: string
+  action: (path: PanelTypes) => void
+}
+
+export default class PanelHeaderSlim extends React.PureComponent<Props> {
```

Comment:
can we use React.memo instead of a Class?

Reply:
Done